website_event_questions_template/models/event.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    @api.model
+    def _default_question_template_id(self):
+        return self.env['ir.values'].get_default(
+            'event.config.settings', 'event_question_template_id')
+
+    event_question_template_id = fields.Many2one(
+        comodel_name='event.question.template',
+        string='Questions Template',
+        default=lambda d: d._default_question_template_id(),
+    )
+
+    def load_question_template(self):
+        # Onchange method is not possible due to
+        # https://github.com/odoo/odoo/issues/20163#issuecomment-340593894
+        self.ensure_one()
+        vals = [(6, 0, [])]
+        if self.question_ids:
+            raise UserError(_(
+                "You can not load a template if there are defined questions"))
+        if self.event_question_template_id:
+            for question in self.event_question_template_id.question_ids:
+                vals.append((0, 0, {
```

Comment:
For making this auto extensible, you should use this code:
```python
for question in self.event_question_template_id.question_ids:
    vals.append((0, 0, question._convert_to_write(question._cache)))
```
Simpler, elegant, and serves for all the new custom fields (but requires the other change I have mentioned).

Reply:
I have an issue... the first is empty from template... mhuuu... I think to research this...

Reply:
Hmm maybe with @pedrobaeza's method you would also copy reserved columns such as `id`, `create_uid`, etc.

I guess the most elegant would be to make those template models have a method that un-templates them, maybe `consolidate()` or whatever. Then you could do here:

```python
self.question_ids = self.event_question_template.question_ids.consolidate()
```

Another option would be to not have separate models. I think that's becoming the trend lately. For instance, they dropped the mass mailing templates and fell back to simply duplicating them. Have you considered this approach? You'd get all possible fields and behavior out of the box, and a simple `.copy()` would do the dirty job...

Reply:
No, `_convert_to_write` doesn't pass that values. I don't get any way what is the problem right now.